store/imagestore/store.go

```diff
@@ -154,68 +153,34 @@ func (s *Store) populateSize() error {
 		}
 		aciSizes[key] = rd
 
-		id, err := s.GetTreeStoreID(key)
-		if err != nil {
-			return err
-		}
-
-		tsR, err := s.treestore.IsRendered(id)
-		if err != nil {
-			return errwrap.Wrap(errors.New("error determining whether the tree store is rendered"), err)
-		}
-		if tsR {
-			// We associate a tree store with an image by writing the image
-			// hash to a file. We need this so we can update the image size
-			// when we remove the rendered tree store.
-			treepath := s.treestore.GetPath(id)
-			if err := ioutil.WriteFile(filepath.Join(treepath, imagefilename), []byte(key), 0644); err != nil {
-				return errwrap.Wrap(errors.New("error writing app treeStoreID"), err)
-			}
-
-			tsSize, err := s.treestore.Size(id)
-			if err != nil {
-				return err
-			}
-
-			tsSizes[key] = tsSize
-		}
 	}
```

Comment:
I removed this since with the decoupling the store cannot call treestore functions (and it's used only on migration from quite old rkt versions). #2968 proposed fix will cover this.